website/docs/r/cbr_zone.html.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cbr_zone"
+description: |-
+  Manages cbr_zone.
+subcategory: "Context Based Restrictions"
+---
+
+# ibm_cbr_zone
+
+Provides a resource for cbr_zone. This allows cbr_zone to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_cbr_zone" "cbr_zone" {
+	name = "Test Zone Resource Config Basic"
+	description = "Test Zone Resource Config Basic"
+	addresses {
+		type = "ipRange"
+		value = "169.23.22.0-169.23.22.255"
+	}
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your resource.
+
+* `addresses` - (Optional, List) The list of addresses in the zone.
```

Comment:
Please update docs according to resource schema..
contexts is marked as Optional in docs but is required in resource schema..

Can you please cross check all other arguments in both the resources?

Reply:
updated and checked